### ISSUE TITLE
How to use "onSelectedItemObjectsChange"

### ISSUE THREAD

**Issue by ghost** [renrizzolo/react-native-sectioned-multi-select#20]
Please guide me how to use onSelectedItemObjectsChange in sectioned multiselect for multiple item selection.

        <SectionedMultiSelect
              items={this.state.cat}
              uniqueKey='id'
              subKey='children'
              selectText='category'
              showDropDowns={true}
              selectChildren={true}
              readOnlyHeadings={true}
              onSelectedItemObjectsChange = {}
              selectedItems={this.state.category}
              removeAllText="Remove All"
              showRemoveAll={true}
          />

**Comment by renrizzolo** [comment]
So, just like `onSelectedItemsChange`, it's a function that you pass in to the component that returns the selected items. But instead of an array of ids, you get the whole objects back.

```
  onSelectedItemObjectsChange = (selectedItemObjects) => {
    this.setState({ selectedItemObjects })
    console.log(selectedItemObjects) // [ 0: { title: 'Apple', id: 10},  1: { ... }]
  }
```

``` 
<SectionedMultiSelect
 onSelectedItemObjectsChange={this.onSelectedItemObjectsChange}
/>

**Comment by karanwadhwa** [comment]
> So, just like `onSelectedItemsChange`, it's a function that you pass in to the component that returns the selected items. But instead of an array of ids, you get the whole objects back.

tried the above, however onSelectedItemsChange is a required prop. why have that when I'm going to use onSelectedItemObjectsChange?

**Comment by trev91** [comment]
I've also attempted using this function as described in your example. I believe something has broke recently that's causing this function to not work as expected. I keep getting a `onSelectedItemsChange` is not a function, error. It is indeed marked as a required prop, so I tested going into your library and making it not required, but then found myself getting into a rabbit hole of other errors. Are you able to use this `onSelectedItemObjectsChange` as you'd expect @renrizzolo ?
